docs/porting/display.rst

```diff
@@ -239,6 +239,36 @@ In full and direct modes, the buffer size should be large enough for the whole s
 As LVGL can not handle fractional width make sure to round the horizontal resolution to 8-
 (For example 90 to 96)
 
+The :cpp:func:`lv_draw_sw_i1_convert_to_vtiled` function is used to convert a draw buffer in I1 color format from a row-wise (htiled) to a column-wise (vtiled) buffer layout.
+This conversion is necessary for certain display controllers that require a different draw buffer mapping. The function assumes that the buffer width and height are rounded to a multiple of 8.
+The bit order of the resulting vtiled buffer can be specified using the `bit_order_lsb` parameter.
+For more details, refer to the implementation in :cpp:func:`lv_draw_sw_i1_convert_to_vtiled` in :file:`src/draw/sw/lv_draw_sw.c`.
+
+To ensure that the redrawn areas start and end on byte boundaries, you can add a rounder callback to your display driver. This callback will round the width and height to the nearest multiple of 8.
+
+Here is an example of how to implement and set a rounder callback:
+
+.. code:: c
+
+    static void my_rounder_cb(lv_event_t *e)
+    {
+        lv_area_t *area = lv_event_get_param(e);
+
+        /* Round the width to the nearest multiple of 8 */
+        area->x1 = (area->x1 & ~0x7);
+        area->x2 = (area->x2 | 0x7);
```

Comment:
This is done internally for I1 [here](https://github.com/lvgl/lvgl/blob/master/src/core/lv_refr.c#L294-L299).